src/core/tsi/alts/handshaker/alts_tsi_handshaker.h

```diff
@@ -38,6 +38,11 @@
 
 const size_t kTsiAltsNumOfPeerProperties = 5;
 
+// Frame size negotiation extends send frame size range to
+// [kMinFrameSize, kMaxFrameSize]
+ABSL_CONST_INIT extern const size_t kMinFrameSize;
```

Comment:
nit: ABSL_CONST_INIT is not commonly used in grpc core yet.

Let's use the following to be consistent with rest of code. You can remove the declaration from .cc.
Also use TsiAlts prefix (which is used as c style namespace).

```
const size_t kTsiAltsMinFrameSize = 16 * 1024;
const size_t kTsiAltsMaxFrameSize = 128 * 1024;
```